odoo/tests/test-uninstalls.py

```diff
@@ -107,6 +112,25 @@ def test_uninstall(args):
         install(args.database, module_id, module_name)
 
 
+def test_scripts(args):
+    """ Tries to launch standalone scripts tagged with @post_testing """
+    # load the registry once for script discovery
+    odoo.registry(args.database)
+
+    # fetch and filter scripts to test
+    to_test = args.test_scripts.split(',')
+    if len(to_test) == 1 and to_test[0] == 'all':
+        funcs = post_testing.registry.values()
+    else:
+        funcs = [v for k, v in post_testing.registry.items() if k in to_test]
+
+    for func in funcs:
+        with odoo.api.Environment.manage():
+            with odoo.registry(args.database).cursor() as cr:
+                env = odoo.api.Environment(cr, odoo.SUPERUSER_ID, {})
+                func(env)
```

Comment:
Maybe log something like the function name(s) that are run.
What about failures?  Should we stop at the first failure?

Reply:
I added a try catch and added some logging akin to the ones for module operations